.github/workflows/releaser.yml

```diff
@@ -216,11 +216,11 @@ jobs:
           body: ${{ steps.summary.outputs.output }}
           prerelease: ${{ needs.version.outputs.pre != '' || needs.version.outputs.dev != '' || needs.version.outputs.local != '' }}
           name: ${{ env.NIGHTLY_RELEASE == 'true' && 'Nightly release' || format('Release v{0}', needs.version.outputs.full) }}
+          # release-files/Parsec_${{ needs.version.outputs.full }}_win_*.exe*
```

Comment:
Why keeping this line?

Reply:
When we would re-enable the upload of windows artifact, I would have to just move the line and un-comment it.